pytest-pixl/src/pytest_pixl/plugin.py

```diff
@@ -75,3 +76,9 @@ def omop_resources() -> Path:
     traversable = resources.files("pytest_pixl").joinpath("data/omop-resources")
     with resources.as_file(traversable) as path:
         return path
+
+
+@pytest.fixture()
+def azure_keyvault() -> MockKeyVault:
```

Comment:
is this fixture used anywhere?

Reply:
Ah yeah good point, this is leftover from an earlier attempt